### ISSUE TITLE
Web GUI for hosting memes

### ISSUE THREAD

**Issue by realhunter7869** [HemantSachdeva/HacktoberFest#134]
I think having a static page to show the jokes posted by users in Repo at random will be great.... Would like to work on same...

**Comment by HemantSachdeva** [comment]
> I think having a static page to show the jokes posted by users in Repo at random will be great.... Would like to work on same...

I appreciate the idea. Do you have experience making a static webpage that renders divs based on an API response?

**Comment by realhunter7869** [comment]
Yes I do have an idea... We can use api.github.com and raw.githubusercontent.com for getting repo related data and readme content added by user...

**Comment by realhunter7869** [comment]
We can also host the page using github pages

**Comment by HemantSachdeva** [comment]
Hosting it won't be an issue. Just the front end and its integration with backend API would play the role. I can design backend API if you or any other contributor could come up.

**Comment by realhunter7869** [comment]
Maybe we won't be needing a backend at all... We will just get the data from github api and put it there

**Comment by HemantSachdeva** [comment]
I had a look at GitHub REST API for getting the content of a GitHub repo. The issue is that it's encoded XD.
Check it out [here](https://docs.github.com/en/rest/repos/contents#get-repository-content) if you find something useful.

**Comment by realhunter7869** [comment]
I am getting a proper json response without encoding. I guess parsing this on standard web page will be easier.

![image](https://user-images.githubusercontent.com/59429408/193327051-284d4576-e6f2-480e-a2f4-40ed6080ff76.png)

**Comment by realhunter7869** [comment]
Should I work on it?

**Comment by HemantSachdeva** [comment]
Yeah sure.
